src/Utils.vala

```diff
@@ -62,9 +62,8 @@ public class Power.Utils {
             total_x_delta = 0.0;
             Power.Services.DeviceManager.get_default ()
                 .change_brightness ((int) Math.round (dir * BRIGHTNESS_STEP));
-            return true;
-        } else {
-            return false;
         }
+
+        return true;
```

Comment:
Should this be `Gdk.EVENT_STOP`?

Reply:
Yeah, it wasn't used anywhere else in the codebase so I kept true/false for consistence but we can change it :)